src/pages/our-principles/index.tsx

```diff
@@ -118,14 +116,16 @@ const OurPrinciples = () => {
                             </StyledFlex>
                             <Flex fd="column" max_width="792px">
                                 <ColorHeader as="h2" color={principle.color}>
-                                    {principle.title}
+                                    <Localize translate_text={principle.title} />
                                 </ColorHeader>
                                 {principle.description.map((description, key) => (
                                     <Flex key={key} fd="column">
```

Comment:
Nitpick: I realize that isn't a part of task but we can improve it a little bit

```suggestion
                                {principle.description.map((description) => (
                                    <Flex key={description.title} fd="column">
```

Reply:
I have changed as advised.